HVZ.Web/Shared/Ui Helpers/GameTagCounter.razor

```
@@ -0,0 +1,56 @@
+﻿@implements IDisposable
+@inject IGameRepo GameRepo
+@inject ILogger<GameTagCounter> Logger
+
+@if (Player is not null && GameId is not null)
+{
+    <span @attributes="@Attributes">@tags</span>
+}
+
+
+@code {
+
+    [Parameter]
+    public string? GameId { get; set; }
+
+    [Parameter]
+    public Player? Player { get; set; }
+
+    [Parameter(CaptureUnmatchedValues = true)]
+    public Dictionary<string, object>? Attributes { get; set; }
+
+    int? tags;
+
+    protected override void OnParametersSet()
+    {
+        if (GameId is null)
+        {
+            Logger.LogError("GameId cannot be null");
+            return;
```

throw an exception of some kind for errors instead of quietly continuing to run